kvstore.sh

```diff
@@ -289,6 +319,28 @@ complete -F _${ns}_kvstore_complete $cmd"
   fi
 }
 
+kvstore_drop () {
+  local ns="$1"
+  [[ -z "$ns" ]] && echo "Missing param: namespace" >&2 && return 1
+  path=$(_kvstore_path "$ns")
+  \rm ${path}
```

Comment:
Why `\rm`?  I haven't seen that expression before

Reply:
Some people alias rm to rm -i and now you have a "do you want to delete this question" pop up in the middle of the command, which may be ok interactive, but not if in a script. the \ means no aliases, no nuttin,

And yet here: https://unix.stackexchange.com/questions/39291/run-a-command-that-is-shadowed-by-an-alias there is no mention of it. Hmm.

I'll investigate the "right" way to do this and amend.